CommunityToolkit.WinUI/Extensions/DispatcherQueueExtensions.cs

```diff
@@ -44,31 +44,32 @@ public static Task EnqueueAsync(this DispatcherQueue dispatcher, Action function
                 }
             }
 
-            static Task TryEnqueueAsync(DispatcherQueue dispatcher, Action function, DispatcherQueuePriority priority)
-            {
-                var taskCompletionSource = new TaskCompletionSource<object?>();
+            var taskCompletionSource = new TaskCompletionSource();
```

Comment:
Is removing the static here an optimization that can also be done for the UWP side? Should that be split as a separate PR so we can better align the two branches?

Reply:
I've actually removed that commit and only left the new functionality in this PR. That should make reviewing this easier as there's no changes to existing code at all now, it's just these new APIs here. If we approve and merge these, then I can work on another follow up PR to integrate these optimizations as the backend of the other existing extensions 😄